src/Components/Notifications/NoticeBoard.tsx

```diff
@@ -55,13 +57,28 @@ export const NoticeBoard: any = () => {
     );
   } else {
     notices = (
-      <Card key="no-notice" className="my-4 rounded-lg">
-        <CardContent>
-          <div className="text-xl text-center semibold">
-            {t("no_notices_for_you")}
+      <div className=" m-auto max-w-sm rounded overflow-hidden shadow-lg ">
+        <img className="w-full" src={bell} alt="Sunset in the mountains"></img>
```

Comment:
Use the actual link to the `img` in the public dir. instead.
```suggestion
        <img className="w-full" src="/images/Notificationbell.png" alt="Sunset in the mountains" />
```